Comanche055/EXECUTIVE.agc

```diff
@@ -350,7 +350,7 @@ WAKETEST	CS	NEWLOC
 		TCF	SPECTEST
 
 # Page 1217
-# PRIORITY CHANGE.  CHANGE THE CONTENTS OF PRIORITY AND SCAN FOR THE JOB OF HIGHEST PRIORITY.
+# PRIORITY CHANGE, CHANGE THE CONTENTS OF PRIORITY AND SCAN FOR THE JOB OF HIGHEST PRIORITY.
```

Comment:
This is a full stop: [p1217](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1217.jpg)

![](https://i.imgur.com/oblLmpS.png)

Reply:
Fixed.